src/pykiso/exceptions.py

```diff
@@ -46,3 +46,17 @@ def __init__(self, aux_name: str) -> None:
         """
         self.message = f"Auxiliary {aux_name} failed at instance creation"
         super().__init__(self.message)
+
+
+class AuxiliaryConnectorRequiredError(PykisoError):
```

Comment:
Wouldn't `ConnectorRequiredError` be self-explaining enough? This seems a bit overkill

Reply:
Done